src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts

```diff
@@ -70,6 +70,7 @@ export class SvgKeyboardKeyComponent implements OnInit, OnChanges, OnDestroy {
     @Input() width: number;
     @Input() keyAction: KeyAction;
     @Input() keybindAnimationEnabled: boolean;
+    @Input() popoverEnabled: boolean;
```

Comment:
I am not so attached to this naming, so it's changed now :smile: But your argument can be span around as well.